### PR TITLE
Enable registration of FCM push tokens separately

### PR DESCRIPTION
**Pull request by hedvigoscar**
Some shortcuts in the interest of time have been made.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -108,8 +108,22 @@
...
+                        assertj-core
+                        </artifactId>
+                </exclusion>
+            </exclusions>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.10.0</version>
```
Comment (johantjellden) [inline pom.xml]: 3.11.0

File: src/main/java/com/hedvig/botService/serviceIntegration/notificationService/NotificationServiceClient.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.hedvig.botService.serviceIntegration.notificationService;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient(name = "notification-service", url = "${hedvig.notificationservice.baseurl}")
```
Comment (johantjellden) [inline src/main/java/com/hedvig/botService/serviceIntegration/notificationService/NotificationServiceClient.java]: set default value with: "${hedvig.notificationservice.baseurl}:notification-service"

File: pom.xml (hunk 1 above)
Comment (meletios) [reply pom.xml]: done

File: src/main/java/com/hedvig/botService/serviceIntegration/notificationService/NotificationServiceClient.java (hunk 2 above)
Comment (meletios) [reply src/main/java/com/hedvig/botService/serviceIntegration/notificationService/NotificationServiceClient.java]: done

**Pull request merged**